### PR TITLE
Add i18n support to contact details component

### PR DESCRIPTION
**Pull request by Shinsina**

### CODE REVIEW

**Comment by Shinsina** [comment]
[How to solve for Mundo Leaders entries in Spanish and English](https://trello.com/c/oKzezqPm/50-how-to-solve-for-mundo-leaders-entries-in-spanish-and-english)

**Comment by Shinsina** [comment]
Build fails as this relies on change found in https://github.com/parameter1/pmmi-media-group-websites/pull/212 (I believe)

File: packages/marko-web-theme-default/components/content/contact-details/index.marko [hunk 1]
```diff
@@ -64,25 +68,25 @@ $ const socialLinkAttrs = getAsObject(input, "socialLinkAttrs");
     <default-theme-content-contact-details-section block-name=blockName>
       <if(content.phone)>
         <div class=`${blockName}__field`>
-          <span class=`${blockName}__label`>Phone:</span>
+          <span class=`${blockName}__label`>${i18n(lang, 'phoneLabel')}:</span>
```
Comment (solocommand) [inline packages/marko-web-theme-default/components/content/contact-details/index.marko]: Use double quotes instead of single quotes in template files

**Review by solocommand — CHANGES_REQUESTED** [review]

File: packages/marko-web-theme-default/components/content/contact-details/index.marko [hunk 2]
```diff
@@ -19,6 +21,8 @@ $ const showFields = fields.some(k => content[k]);
 $ const websiteLinkAttrs = getAsObject(input, "websiteLinkAttrs");
 $ const socialLinkAttrs = getAsObject(input, "socialLinkAttrs");
 
+$ const lang = input.lang || 'en'
```
Comment (solocommand) [inline packages/marko-web-theme-default/components/content/contact-details/index.marko]: This one too

**Review by solocommand — CHANGES_REQUESTED** [review]

File: packages/marko-web-theme-default/i18n.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const translations = require('./translations.js');
+
+export default function i18n(lang, fieldLabelKey) {
```
Comment (zarathustra323) [inline packages/marko-web-theme-default/i18n.js]: This should use CommonJS modules (e.g. `module.exports`) not ESM (`export default`)

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Review by solocommand — APPROVED** [review]

**Pull request merged**